File: 20251108_Contenido_info.js
/*
Cómo evaluar la longitud de cadenas de caracteres en JavaScript:

Para evaluar la longitud de cadenas de caracteres, se puede utilizar la propiedad length. Considera el siguiente ejemplo. Creamos una función personalizada que acepta una cadena de caracteres y por medio de sentencias condicionales, evaluamos la condición que le corresponde y retornamos el mensaje apropiado. Después simplemente lo mandamos a la consola.
*/

function publicacion(mensaje){
    if (mensaje.length <= 40) {
        return "publicación corta";
    } else if (mensaje.length < 80) {
        return "publicación larga";
    } else {
        return "publicación inválida";
    }
}

console.log(publicacion("Hola mundo"))

File: 20251108_Contenido_info.py
"""
Cómo evaluar la longitud de cadenas de caracteres en Python:

Para evaluar la longitud de cadenas de caracteres, se puede utlizar la función len. Considera el siguiente ejemplo, creamos una función personalizada que acepta una cadena de caracteres y por medio de sentencias condicionales, evaluamos la condición que le corresponde y retornamos el mensaje apropiado. Después simplemente imprimimos.
"""

def publicacion(mensaje):
    if len(mensaje) <= 40:
        return "publicación corta"
    elif len(mensaje) < 80:
        return "publicación larga"
    else:
        return "publicación inválida"

print(publicacion("Hola mundo"))
